packages/generators/src/handlers/default.ts

```diff
@@ -0,0 +1,124 @@
+
+/**
+ * Asks questions to the user used to modify generation
+ * @param self Generator values
+ * @param Question Contains questions
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function questions(self: CustomGenerator, Question: Record<string, any>): Promise<void> {
+    // Handle JS language solutions
+    const { jsLang } = await Question.List(
+        self,
+        'jsLang',
+        'Which of the following JS solutions do you want to use?',
+        ['none', 'ES6', 'Typescript'],
+        'none',
+        self.useDefaults,
+    );
+
+    switch (jsLang) {
+        case 'ES6':
+            self.dependencies = [...self.dependencies, 'babel-loader', '@babel/core', '@babel/preset-env'];
+            break;
+        case 'Typescript':
+            self.dependencies = [...self.dependencies, 'typescript', 'ts-loader'];
+            break;
+    }
+
+    // Configure devServer configuraion
+    const { devServer } = await Question.Confirm(self, 'devServer', 'Do you want to webpack-dev-server?', true, self.useDefaults);
+    if (devServer) {
+        self.dependencies = [...self.dependencies, 'webpack-dev-server'];
+    }
+
+    // Handle addition of html-webpack-plugin
+    const { htmlWebpackPlugin } = await Question.Confirm(
+        self,
+        'htmlWebpackPlugin',
+        'Do you want to simplify the creation of HTML files for your bundle?',
+        true,
+        self.useDefaults,
+    );
+    if (htmlWebpackPlugin) {
+        self.dependencies = [...self.dependencies, 'html-webpack-plugin'];
+    }
+
+    // Handle CSS solutions
+    const { cssType } = await Question.List(
+        self,
+        'cssType',
+        'Which of the following CSS solutions do you want to use?',
+        ['none', 'CSS'],
+        'none',
+        self.useDefaults,
+    );
+
+    switch (cssType) {
+        case 'CSS':
+            self.dependencies = [...self.dependencies, 'style-loader', 'css-loader'];
+            break;
+    }
+
+    // store all answers for generation
+    self.answers = { ...self.answers, jsLang, devServer, htmlWebpackPlugin, cssType };
+}
+
+/**
+ * Handles generation of project files
+ * @param self Generator values
+ */
+export function generate(self: CustomGenerator): void {
+    self.fs.extendJSON(
+        self.destinationPath('package.json'),
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        require(resolveFile('package.json.js'))(self.answers.devServer),
+    );
+
+    const generateEntryFile = (entryPath: string): void => {
+        entryPath = entryPath.replace(/'/g, '');
+        if (self.answers.jsLang == 'Typescript') {
+            entryPath += 'ts';
+        } else {
+            entryPath += 'js';
+        }
+        self.fs.copyTpl(resolveFile('index.js'), self.destinationPath(entryPath));
+    };
+
+    // Generate entry file/files
+    const entry = './src/index.';
+    if (typeof entry === 'string') {
+        generateEntryFile(entry);
+    }
```

Comment:
Are you going to add the prompt for multiple bundles?

Reply:
Not for now. Will update this logic.